runtime/src/lib.rs

```diff
@@ -863,6 +873,67 @@ impl pallet_verifiers::Config<pallet_proofofsql_verifier::ProofOfSql<Runtime>> f
     >;
 }
 
+parameter_types! {
+    pub const Coprocessor: Option<StateMachine> = Some(StateMachine::Kusama(4009));
+    pub const HostStateMachine: StateMachine = StateMachine::Substrate(*b"zkv_");
+}
+
+impl pallet_ismp::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type AdminOrigin = EnsureRoot<AccountId>;
+    type HostStateMachine = HostStateMachine;
+    type TimestampProvider = Timestamp;
+    // TODO: Update to actual currency once stable coin on relayer is established
+    type Currency = NullCurrency;
```

Comment:
Is it required even if payments happen off-chain ? We don't envision integrating a stable coin for now